### ISSUE TITLE
SPIKE: USWDS and Node Upgrade 

### ISSUE THREAD

**Issue by MaeganWood64** [US-EPA-CAMD/PI-1#3476]
- USWDS 3.0 and Node 18 have been released, current versions used across CAMPD and ECMPS will potentially eb depcreciated. 
- Research which versions will be implemented across apps in IPI 8 &P

**Comment by MaeganWood64** [comment]
- [ ]  Moved down in priority, but waiting on Trustworks shifting to USWDS 3.0, and when cloud.gov starts supporting node 18

**Comment by MaeganWood64** [comment]
Ticket no longer needed
